backend/cellguide/api/v1/upload/actions.py

```diff
@@ -0,0 +1,48 @@
+import json
+import os
+import re
+
+import requests
+from flask import jsonify, make_response
+
+from backend.common.utils.http_exceptions import ForbiddenHTTPException
+from backend.layers.auth.user_info import UserInfo
+from backend.layers.thirdparty.s3_provider import S3Provider
+
+
+def post(body: dict, token_info: dict):
+    try:
+        UserInfo(token_info).is_cxg_admin()
+    except Exception as e:
+        raise ForbiddenHTTPException("Unauthorized") from e
+
+    cell_onthology_id = body["cell_onthology_id"]
```

Comment:
`cell_ontology_term_id` is the nomenclature used throughout the portal. let's use it to be consistent